File: BookingServices/CusromerProject/DTO/Services/ServiceDetails.cs
namespace CusromerProject.DTO.Services
{
    public class ServiceDetails
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Details { get; set; }
        public string Location { get; set; }
        public TimeSpan StartTime { get; set; }
        public TimeSpan EndTime { get; set; }
        public int Quantity { get; set; }
        public decimal InitialPayment { get; set; }
        public string Category { get; set; }
        public string ProviderName { get; set; }
        public decimal PriceForTheCurrentDay { get; set; }

        public List<string> Images { get; set; }
        public List<AllServicesDetails> RelatedServices { get; set; }
    }
}

File: CustomerAngularProject/housekey/src/app/common/interfaces/ServiceDetails.ts
import { provider } from "./provider";
import { Service } from "./service";
export interface ServiceDetails {
    id: number;
    name: string;
    details: string;
    location: string;
    startTime: string;
    endTime: string;
    providerName: string;
    priceForTheCurrentDay: number;
    category: string;
    images: string[];
    quantity: number;
    relatedServices: Service[];
    initialPayment: number;
    provider: provider;
}
